hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClientConnectionManagerImpl.java

```diff
@@ -216,7 +216,15 @@ public ClientConnection getConnection(Address target) {
         if (target == null) {
             return null;
         }
-        return connections.get(target);
+        ClientConnection connection = connections.get(target);
```

Comment:
can you add a line of comment on the master branch? This way we know you have not turned to the dark side.